feature-libs/product/configurators/common/components/conflict-suggestion/configurator-conflict-suggestion.component.html

```diff
@@ -1,9 +1,9 @@
 <div
   *ngIf="displayConflictSuggestion(currentGroup)"
-  class="cx-configurator-conflict-suggestion"
+  class="cx-suggestion-msg"
   tabindex="0"
 >
```

Comment:
Just FYI we could drop the wrapper `<div>`, by:
- using `<ng-container>` with ngIf instead
- add line `@HostBinding('tabindex') get tabindex() { return '0' };` in the component TS controller

(note: I recall there are other simlar places in this PR)

Reply:
Both points were build in ;)
I go through all refactored components and add this line "_@HostBinding('tabindex') get tabindex() { return '0' }; in the component TS controller_"

Reply:
Done where it was so far possible!